pallets/api/src/messaging/benchmarking.rs

```diff
@@ -0,0 +1,64 @@
+//! Benchmarking setup for pallet_api::nonfungibles
+#![cfg(feature = "runtime-benchmarks")]
+
+use frame_benchmarking::{account, v2::*};
+use frame_support::{dispatch::RawOrigin, traits::Currency, BoundedVec};
+use sp_runtime::traits::Zero;
+
+use super::*;
+use crate::Read as _;
+const SEED: u32 = 1;
+
+// See if `generic_event` has been emitted.
+fn assert_has_event<T: Config>(generic_event: <T as crate::messaging::Config>::RuntimeEvent) {
+	frame_system::Pallet::<T>::assert_has_event(generic_event.into());
+}
+
+#[benchmarks(
+	where
+	T: pallet_balances::Config
+)]
+mod messaging_benchmarks {
+	use super::*;
+
+	#[benchmark]
+	// x: The number of removals required.
+	fn remove(x: Linear<1, 255>) {
```

Comment:
is it possible to use `T::MaxRemovals` here? (instead of `255`)

Reply:
No but im tempted to increase the limit, what do you think?

Reply:
Hmm, 255 seems like a lot of messages. Maybe we can keep it there for now.

Instead of 255 directly, would it be possible to add a config (const) item to the mock and then use that instead?